Tasks/Common/kubernetes-common-v2/package-lock.json

```diff
@@ -25,6 +35,31 @@
         "uuid": "^3.0.1"
       }
     },
+    "azure-pipelines-tool-lib": {
+      "version": "0.11.0",
```

Comment:
Why aren't you using the latest one?

Reply:
I looked to DockerInstallerV0 task which works Ok with proxy, and uses v 0.11.0 so . . . I thought it was a good starting point, at least. :). Anyway I've just upgrade the version to 0.12.0